amazon-kinesis-client/src/test/java/software/amazon/kinesis/common/RecordValidatorQueueTest.java

```diff
@@ -0,0 +1,55 @@
+package software.amazon.kinesis.common;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.mockito.Mock;
+import software.amazon.kinesis.utils.RecordValidatorQueue;
+
+public class RecordValidatorQueueTest {
+
+    @Mock
+    private RecordValidatorQueue recordValidator;
+
+    private static final String shardId = "ABC";
+
+    private final int outOfOrderError = -1;
+    private final int missingRecordError = -2;
+
+    private final int noError = 0;
+
+    @Test
+    public void validationFailedRecordOutOfOrderTest() {
+        recordValidator = new RecordValidatorQueue();
+        recordValidator.add(shardId, "0");
+        recordValidator.add(shardId, "1");
+        recordValidator.add(shardId, "3");
+        recordValidator.add(shardId, "2");
+
+        int error = recordValidator.validateRecords( 4 );
```

Comment:
I believe we don't need space between 4 and ( 
Can check with other places for consistency.